### ISSUE TITLE
Content Manager

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#1782]
Content Manager<br>
http://ift.tt/2nDhqmt<br><p><img alt="Resized_logo" src="http://ift.tt/2mYfFmM"></p>
<p><strong>Headquarters:</strong> Lisbon, Portgual<br><strong>URL:</strong> <a href="http://ift.tt/2nrbrlt">http://ift.tt/2nrbrlt</a></p>
<div>
<strong>What we Do</strong><br>
</div>
<div><strong><br></strong></div>
<div>At The Mimic method, we create products for people to learn languages online. But our approach is different from that of other language companies.&nbsp;We don&rsquo;t teach literacy, grammar theory, or translation like everyone else.</div>
<div>Instead, we help people unblock their hearing and pronunciation in the language.</div>
<div class="paragraph_break"><br></div>
<ul>
<li>
<em><em>When you unblock your hearing, you can&nbsp;</em></em><em>understand</em><em><em>&nbsp;native speakers when they talk fast.</em></em>
</li>
<li>
<em><em>When you unblock your pronunciation, you can&nbsp;</em></em><em>impress</em><em><em>&nbsp;native speakers with your accent.</em></em>
</li>
</ul><div><em><br></em></div>
<div>The more you can understand and impress people in conversation, the more you can connect with them.&nbsp;We don&rsquo;t care about people reading foreign language newspapers or passing language certification exams.</div>
<div>We care about people making human connections across the language barrier.</div>
<div class="paragraph_break"><br></div>
<div>We believe the world would be a better place if more people did this. There would be less fear and violence and more love and creativity.&nbsp;If you agree with our philosophy, and you want to help us in our mission, then we may have the perfect job for you&hellip;</div>
<div class="paragraph_break"><br></div>
<div><strong>The Position &ndash; Content Czar</strong></div>
<div>To promote our brand, we need to produce content that is&nbsp;educational, entertaining and original.&nbsp;The more articles, videos and graphics we produce, the more visitors we bring to the site.</div>
<div>As Content Czar, you will start by creating most of the content yourself.</div>
<div class="paragraph_break"><br></div>
<div>This is the fastest way to familiarize yourself with our learning philosophy and voice. After initial onboarding, expect to produce articles each&nbsp;week to start.&nbsp;You will also test different marketing strategies while communicating directly with our audience through email and social media.</div>
<div>Speaking directly with readers and customers is the fastest way to develop your marketing intuition.</div>
<div class="paragraph_break"><br></div>
<div>Then soon as you have a good feel for what works, you will help build our &ldquo;content machine.&rdquo;</div>
<div class="paragraph_break"><br></div>
<div>This involves formalizing the processes for content research, production and publication. It also involves recruiting, training and managing other people to operate the machine.</div>
<div class="paragraph_break"><br></div>
<div>In the long run, you will focus less on content creation and more on strategy and leadership.</div>
<div class="paragraph_break"><br></div>
<div><strong>Scope of Work (a day in the life)</strong></div>
<ul>
<li>Full Time&nbsp;position</li>
<li>&ldquo;High-Touch,&rdquo; calls multiple times per week</li>
<li>Manage&nbsp;blog content and guest post outreach</li>
<li>Produce/Edit 2-3 blog content per week</li>
<li>Doing research within the audience and tailoring content according to their needs</li>
<li>Coordinate with writers, designers and other freelancers</li>
<li>Keeping up with the SEO industry at least on the surface</li>
</ul><div class="paragraph_break"><br></div>
<div><strong>Compensation and Career</strong></div>
<div>We are looking for an&nbsp;Intrapreneurial-type person to grow with our company in the long term.</div>
<div class="paragraph_break"><br></div>
<div>Currently, we work as a distributed team. That means you can work from anywhere in the world on your own schedule. All that we require is that you are very reachable and deliver results.</div>
<div class="paragraph_break"><br></div>
<div>During this&nbsp;6-month trial period, we will pay you $2,000 per month. If the trial goes well, we will negotiate a higher full-time salary based on your performance.</div>
<div class="paragraph_break"><br></div>
<div>In the long term, your salary will increase as the company expands. And if you prove yourself to be a rockstar, there is a potential for commissions and/or profit share.</div>
<div class="paragraph_break"><br></div>
<div><strong>Learning and Growth</strong></div>
<div>Learning and Growth are two of our core values at The Mimic Method. One way that we live these&nbsp;values is by investing heavily in our&nbsp;own continued education.</div>
<div class="paragraph_break"><br></div>
<div>If we select you for this position, we will provide you with anything you need to make you a more effective Content Czar. This means professional training in highly useful skills&nbsp;like&nbsp;digital marketing, copywriting, and personnel&nbsp;management, etc.</div>
<div class="paragraph_break"><br></div>
<div>There is no better way for you to develop than to study the most effective strategies out there and apply them to a real world setting.</div>
<div class="paragraph_break"><br></div>
<div>We are looking for someone who values growth and learning above all else &ndash; and we plan to nurture this value.</div>
<div class="paragraph_break"><br></div>
<div><strong>Must-have Qualities</strong></div>
<div>For this position, we are looking for someone with a specific set of qualities. Since we know these qualities to be rare, most people reading this will NOT have them all.</div>
<div class="paragraph_break"><br></div>
<div>Please do not apply for this job unless you have ALL the qualities described below.</div>
<div class="paragraph_break"><br></div>
<ul>
<li>Values-Aligned</li>
<li>Shrewd/Intellectual</li>
<li>Well-spoken/well-written</li>
<li>Highly-structured</li>
<li>Results-Driven</li>
</ul><div class="paragraph_break"><br></div>
<div>Allow me to briefly touch upon each trait.</div>
<div class="paragraph_break"><br></div>
<div><em>Values-Aligned</em></div>
<div>To work with us long term, you need to share our core values of Truth, Growth, Education and Respect.</div>
<div class="paragraph_break"><br></div>
<div>If you strongly identify with the statements below, then you probably share our values:</div>
<div class="paragraph_break"><br></div>
<ul>
<li>I must constantly grow as a person, otherwise I am dying.</li>
<li>I have an insatiable appetite for knowledge, and I will continue learning for the rest of my life.</li>
<li>My sex, gender, race, ethnicity, sexual orientation, nationality, etc, doesn&rsquo;t make me inherently superior to anyone, which is why I make a point to respect everyone.</li>
<li>Before I die, I must travel the world and know its people as much as I can.</li>
<li>Before I die, I must create or contribute to something meaningful.</li>
</ul><div>Do you share these values? If so, keep reading!</div>
<div class="paragraph_break"><br></div>
<div><em>Introspective/Shrewd</em></div>
<div>Are you the introspective type? Do you spend way too much time contemplating the nature of society, the cosmos, or the inner-workings of your own mind?</div>
<div class="paragraph_break"><br></div>
<div>I know I do.</div>
<div class="paragraph_break"><br></div>
<div>As content manager, you and I will speak on a regular basis to develop content and strategy together.</div>
<div class="paragraph_break"><br></div>
<div>If you are a fellow introspective, then our conversations will energize us both. This energy will transfer to our work and enable us to reach our goals faster.</div>
<div class="paragraph_break"><br></div>
<div>But I don&rsquo;t want someone who thinks like me. I&rsquo;m a bit of a scatter-brain, so I need an unscattered brain to complement my thinking. &nbsp;<br>
</div>
<ul>
<li>Have you always had an intuitive understanding of people&rsquo;s thoughts and emotions?</li>
<li>Do you often know what people are going to say before they say it?</li>
<li>Has someone ever told you &ldquo;I feel like you know me better than I know myself!&rdquo;</li>
</ul><div class="paragraph_break"><br></div>
<div>If so, then you might be exactly the person I&rsquo;m looking for.</div>
<div class="paragraph_break"><br></div>
<div><em>Well Spoken/Well Written</em></div>
<div>My preferred style of communication is live conversation. To refine my ideas, I need to stress-test them in debate.&nbsp;Are you able to formulate your views in the moment and present them with clear argumentation?&nbsp;If so, then I&rsquo;m looking forward to our first conversation!</div>
<div class="paragraph_break"><br></div>
<div>But more important than that, I need someone with superb writing skills.</div>
<div class="paragraph_break"><br></div>
<div>You need to take complex subjects and break them down into simple explanations. Your writing must be clear but also enjoyable to read. Bonus points if you can spice it up with some humor!</div>
<div class="paragraph_break"><br></div>
<div>Your writing ability will be the main thing that we&rsquo;ll test you on during the interview process.</div>
<div class="paragraph_break"><br></div>
<div><em>Highly Structured</em></div>
<div>Most people love the idea of working on their own schedule, but few people are actually cut out for it.</div>
<div class="paragraph_break"><br></div>
<div>Getting things done on a consistent basis without a rigid external structure requires discipline.</div>
<div class="paragraph_break"><br></div>
<div>For this role, we need someone who&rsquo;s always had that sort of discipline. The type of person who likes to organize work schedules, computer files and sock drawers for the fun of it.</div>
<div class="paragraph_break"><br></div>
<div>If you&rsquo;re the right person, then you know exactly what I&rsquo;m talking about!</div>
<div class="paragraph_break"><br></div>
<div><em>Results Driven</em></div>
<div>As fun and interesting as all this language stuff may be, at the end of the day, we want results.</div>
<div class="paragraph_break"><br></div>
<div>As content manager, your chief aim is to drive traffic to our site.</div>
<div class="paragraph_break"><br></div>
<div>We we will assess your performance based on hard metrics like views and shares. So those of you who are numerophobic need not apply.</div>
<div class="paragraph_break"><br></div>
<div>We are looking for someone with grit; someone who won&rsquo;t stop working at something until she gets the results she&rsquo;s looking for.</div>
<div class="paragraph_break"><br></div>
<div><strong>Nice-to-Have Qualities</strong></div>
<div>The following qualities are not required for the application, but they are helpful. If you have any of them, be sure to mention in your application video:</div>
<div class="paragraph_break"><br></div>
<ul>
<li>Can speak multiple languages</li>
<li>Has learned 1 or more foreign language to fluency as an adult</li>
<li>Is willing to spend extensive time in Lisbon, Portugal (company HQ)</li>
<li>Can play a musical instrument or can sing</li>
<li>Has extensive experience writing for other publications.</li>
<li>Has maintained a personal blog or vlog for extended period of time.</li>
<li>Knows how to write SEO-friendly articles</li>
<li>Has experience in online marketing (Content, Social Media, PPC, etc.)</li>
<li>Has experience managing personel.</li>
<li>Has experience in online business.</li>
</ul><div class="paragraph_break"><br></div>
<div><strong>Application Process</strong></div>
<div><a href="http://ift.tt/2mYlC30">Apply Here!</a></div>
<div class="paragraph_break"><br></div>
<div>There will be 3 steps to this selection process:</div>
<div class="paragraph_break"><br></div>
<ol>
<li>Writing test and video application</li>
<li>Interview round 1 and Video Test</li>
<li>Interview round 2</li>
</ol><div class="paragraph_break"><br></div>
<div><strong>Writing Test and Video Application</strong></div>
<div>The first step is to test your ability to translate my random thoughts into clear prose.</div>
<div class="paragraph_break"><br></div>
<div>In the video below, I explain the importance of memorizing scripts for language-learning. At the moment of recording, I improvised a metaphor about swimming that turned out much less clear than I hoped.</div>
<div class="paragraph_break"><br></div>
<div><em>Writing Test (Start video at 4:30)</em></div>
<div><a href="https://youtu.be/5WOO-P6evis">https://youtu.be/5WOO-P6evis</a></div>
<div class="paragraph_break"><br></div>
<div>Your job is to figure out what my point was (if I ever had one) and write it out clearly and convincingly in a blog post. You may use as few or as many words as you deem necessary.</div>
<div class="paragraph_break"><br></div>
<div>The main goal is to have a quality piece of content that captures the essence of what I was trying to communicate.</div>
<div class="paragraph_break"><br></div>
<div><em>Video Application</em></div>
<div>Next, you must record a video of yourself explaining two things:</div>
<div class="paragraph_break"><br></div>
<ul>
<li>Why do you want this position?</li>
<li>Why should we want you?</li>
</ul><div class="paragraph_break"><br></div>
<div>The video should be between 3-10 minutes in length and uploaded to a video-hosting service like vimeo or youtube.</div>
<div class="paragraph_break"><br></div>
<div>Once your video and article are ready, fill out&nbsp;<a href="http://ift.tt/2mYlC30">this application form</a>. In the form, you will be able to attach your test article as a PDF and application video as a URL link.</div>
<div class="paragraph_break"><br></div>
<div><strong>Interview Round 1 &amp; Test</strong></div>
<div>If I like your video and writing sample, I will contact you to set up a skype interview.</div>
<div class="paragraph_break"><br></div>
<div>At the end of this interview, I will explain a phonetic concept to you. Then it will be your job to create a short video explanation of that concept.</div>
<div class="paragraph_break"><br></div>
<div>The goal here is to test your versatility as an educator and content creator.</div>
<div class="paragraph_break"><br></div>
<div><strong>Interview Round 2</strong></div>
<div>If you make it past the video test, we will have a final interview with you and the rest of the team. This is where we ask the tough questions to weed out the final winner.</div>
<div class="paragraph_break"><br></div>
<div>&mdash;</div>
<div class="paragraph_break"><br></div>
<div>Does this description fit you? Are you game for some unconventional interviewing practices?</div>
<div class="paragraph_break"><br></div>
<div>Do you want to start a new location-independent career, build something epic, grow professionally, and potentially get rich in the process?</div>
<div class="paragraph_break"><br></div>
<div><a href="http://ift.tt/2mYlC30">Then send us an application ASAP!</a></div>
<p><strong>To apply:</strong> (Please read the job post first before applying) <a href="http://ift.tt/2nrbrlt">http://ift.tt/2nrbrlt</a> Then, Apply Here: <a href="http://ift.tt/2mYlC30">http://ift.tt/2mYlC30</a></p>
<br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
March 15, 2017 at 12:25AM
